cumulusci/cli/cci.py

```diff
@@ -1053,6 +1053,41 @@ def org_list(runtime, plain):
     persistent_table.echo(plain)
 
 
+@org.command(name="prune", help="Removes all expired orgs from the current project")
+@pass_runtime(require_project=False, require_keychain=True)
+def org_prune(runtime):
+    org_configs = {
+        org: runtime.keychain.get_org(org) for org in runtime.keychain.list_orgs()
+    }
+    scratch_configs = getattr(runtime.project_config, "orgs__scratch", {})
+
+    expired_orgs_removed = []
+    org_shapes_skipped = []
+    active_orgs_skipped = []
+    for org, org_config in org_configs.items():
+        if scratch_configs.get(org):
+            org_shapes_skipped.append(org)
+        elif org_config.active:
+            active_orgs_skipped.append(org)
+        elif isinstance(org_config, ScratchOrgConfig) and not org_config.active:
+            try:
+                runtime.keychain.remove_org(org)
+                expired_orgs_removed.append(org)
+            except Exception as e:
+                click.echo(e)
```

Comment:
I think these 2 lines are unnecessary: if there's an exception it will fall through to the generic exception handler which also prints the exception (as well as making sure the traceback gets logged to where `cci error info` can find it).